gpMgmt/bin/gpcheckperf

```diff
@@ -267,6 +270,12 @@ def parseCommandLine():
         GV.opt['--duration'] = 15
         print '[INFO] Invalid network duration specified.  Using default (15 seconds)'
 
+    if GV.opt['--buffer-size'] > 0 and GV.opt['--netclient'].find('netperf') >= 0:
+        print '[Warning] Applying the --buffer-size option is not possible when the --netperf option is enabled.'
+
+    if GV.opt['--buffer-size'] == 0:
```

Comment:
then it is not needed

Reply:
It is needed for check upper, to check for entered --buffer-size along with --netperf.

Reply:
I mean check `if GV.opt['--buffer-size'] == 0:`

Reply:
Before running gpcheckperf we should check if `--buffer-size` is being used at the same time with `--netperf` and if so we should print a warning to the log. If you initially set `--buffer-size` to 8, then it will be impossible to understand if this is the default value or the user entered the same in the attribute.